.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -65,7 +65,7 @@ body:
 
   - type: textarea
     attributes:
-      label: How does Datadog Help You
+      label: How Does Datadog Help You?
```

Minor: I Realize Every Other Label Is Camel Cased But Is It Just Me Or Does It Sound Really Awkward In Longer Sentences Such As This One?

```suggestion
      label: How does Datadog help you?
```

---

Let's try it.